libmscore/undo.h

```diff
@@ -1317,5 +1317,27 @@ class FretMarker : public UndoCommand {
       UNDO_NAME("FretMarker")
       };
 
+//---------------------------------------------------------
+//   MoveTremolo
+//---------------------------------------------------------
+
+class MoveTremolo : public UndoCommand {
+      Score* score;
+      Fraction chord1Tick;
+      Fraction chord2Tick;
+      Tremolo* trem;
+      int track;
+
+      Chord* oldC1;
+      Chord* oldC2;
+
+      void undo(EditData*) override;
+      void redo(EditData*) override;
+
+   public:
+      MoveTremolo(Score* s, Fraction c1, Fraction c2, Tremolo* tr, int t) : score(s), chord1Tick(c1), chord2Tick(c2), trem(tr), track(t) {}
```

Comment:
It is better to extract constructor as a method to .cpp file. Assigning so many variables litters the code in .h file.

Reply:
I mean, most of the constructors for undo are in the .h file. But sure, I'll move it if you want.

Reply:
Please can I keep it like this? Literally almost every other constructor is in undo.h, a lot with 4+ variables.